feature/metis/shared/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/shared/db/entities/MetisPostContextEntity.kt

```diff
@@ -14,7 +14,7 @@ import androidx.room.Index
  */
 @Entity(
     tableName = "metis_post_context",
-    primaryKeys = ["client_post_id", "course_id", "conversation_id", "type", "server_id"],
```

Comment:
Why is the server id here not needed?

Reply:
Hey, it used to be like this pre hotfix. As far as I know from reading through comments in the code the server_id could be null in an offline-state.